packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.tsx

```diff
@@ -53,7 +53,7 @@ export const ButtonCascader: React.FC<ButtonCascaderProps> = props => {
       {...rest}
       expandIcon={null}
     >
-      <button className={cx('gf-form-label', className)} disabled={props.disabled}>
+      <button className={cx('gf-form-label', className)} disabled={props.disabled} data-testid="bc-button">
```

Comment:
I think it's better to use `aria-label` so we can reuse the selector in e2e tests.